### DISCUSSION TITLE
Different states for button interaction and complex checklist usage

### DISCUSSION THREAD

**Discussion by mindytom** [twilio-labs/paste#1309] [Office hours] [answered]
Hey DSYS!

I have a few questions regarding different components and interactions for Event Streams. 

1. Would like to get the team's input on the different states of this button interaction. I was following [#1291 discussion](https://github.com/twilio-labs/paste/discussions/1291) as it's a similar topic and wanted to see if there's any accessibility concerns with how I'm using it. Is this a pattern used anywhere in Console today? 
![ARN Input States](https://user-images.githubusercontent.com/65037539/112230031-5f77d480-8bf1-11eb-8d71-b584afbc86d8.png)

2. Discuss [#1271](https://github.com/twilio-labs/paste/discussions/1271) in person. Here is another concept I'm exploring that may help with the v1, v2, v3. 
<img width="1269" alt="Screen Shot 2021-03-23 at 4 12 41 PM" src="https://user-images.githubusercontent.com/65037539/112230738-9ac6d300-8bf2-11eb-9e04-2fe4f258c8f3.png">

**Comment by gloriliale** [comment]
Hi @mindytom! Thanks for posting this so that we can discuss the details in office hours!

**Comment by nkrantz** [comment, marked as answer]
Hi @mindytom! Please find the recording from yesterday's office hours [here](https://www.loom.com/share/7e2e180aa092438fbc9ac570f710fb8a) (password is in #help-design-system on slack).
